### PR TITLE
Added https://github.com/r-caamano/ebpf-tproxy-splicer to list of exa…

### PR DESCRIPTION
**Pull request by r-caamano**
Would like to add my project to your list of eBPF examples.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [ebpf-samples](https://github.com/vbpf/ebpf-samples) - A collection of compiled (as ELF object files) samples gathered from several projects, primarily intended to serve as test cases for user space verifiers.
 - [ebpf-kill-example](https://github.com/niclashedam/ebpf-kill-example) - A fully documented and tested example of an eBPF probe that logs all force-kills and prints them out in user-space.
 - [redbpf examples](https://github.com/foniod/redbpf/tree/main/examples) - Example programs for using RedBPF to write eBPF programs in Rust.
-
+- [TC-eBPF example](https://github.com/r-caamano/ebpf-tproxy-splicer) - Program uses TC-eBPF to provide statefull Firewalling and socket redirection.
```
Comment (qmonnet) [inline readme.md:195]: Please re-add the empty line after your entry.

**Review by qmonnet — CHANGES_REQUESTED** [review]
Sorry I missed this PR. I see the project have moved, since. If you're still interested, could you please update your entry?

File: readme.md (hunk 1 above)
Comment (r-caamano) [reply readme.md]: Hi sorry,  I had not seen the changes requested.  I am still interested.  Note: I added the missing line.  Also I moved the project so the URL is also changed.

**Review by qmonnet — APPROVED** [review]
Looks better now, thank you!

File: readme.md (hunk 1 above)
Comment (qmonnet) [reply readme.md]: Happens to me all the time, no worries! Thanks for addressing the feedback.

**Pull request merged**